### PR TITLE
LCAM-913

### PR DESCRIPTION
**Pull request by skasthuri36326**
## What

[LCAM-913](https://dsdmoj.atlassian.net/browse/LCAM-913)

Describe what you did and why.
- CI/CD pipeline set up for MAAT Orchestration service

## Checklist

Before you ask people to review this PR:

- [ ] Tests should be passing: `./gradlew test`
- [ ] Github should not be reporting conflicts; you should have recently run `git rebase main`.
- [ ] Avoid mixing whitespace changes with code changes in the same commit. These make diffs harder to read and conflicts more likely.
- [ ] You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- [ ] You should have checked that the commit messages say why the change was made.

[LCAM-913]: https://dsdmoj.atlassian.net/browse/LCAM-913?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: maat-orchestration/docker-compose.yml [hunk 1]
```diff
@@ -10,3 +10,16 @@ services:
     depends_on:
       postgres:
         condition: service_healthy
+
+  postgres:
```
Comment (venkat980) [inline maat-orchestration/docker-compose.yml:14]: This service doesn't require postgres

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+            --build-arg APP_BRANCH=${CIRCLE_BRANCH} \
+            --build-arg BUILD_DATE=$(date +%Y-%m-%dT%H:%M:%S%z) \
+            -t $AWS_ECR_ACCOUNT_URL/laa-crime-apps-team/laa-maat-orchestration-dev-ecr:${CIRCLE_SHA1} .
+      - snyk/scan:
+          docker-image-name: "$AWS_ECR_ACCOUNT_URL/laa-crime-apps-team/laa-maat-orchestration-dev-ecr:${CIRCLE_SHA1}"
+          <<: *snyk_options
+      - run:
+          name: Push to ECR
+          command: |
+            docker push "$AWS_ECR_ACCOUNT_URL/laa-crime-apps-team/laa-maat-orchestration-dev-ecr:${CIRCLE_SHA1}"
```
Comment (venkat980) [inline .circleci/config.yml:141]: Please chekc this, it should be $AWS_ECR_ACCOUNT_URL/laa-crime-apps-team/**laa-maat-orchestration-dev**

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+          name: Build docker image
+          command: |
+            docker build \
+            --build-arg COMMIT_ID=${CIRCLE_SHA1} \
+            --build-arg BUILD_TAG=${CIRCLE_SHA1} \
+            --build-arg APP_BRANCH=${CIRCLE_BRANCH} \
+            --build-arg BUILD_DATE=$(date +%Y-%m-%dT%H:%M:%S%z) \
+            -t $AWS_ECR_ACCOUNT_URL/laa-crime-apps-team/laa-maat-orchestration-dev-ecr:${CIRCLE_SHA1} .
+      - snyk/scan:
+          docker-image-name: "$AWS_ECR_ACCOUNT_URL/laa-crime-apps-team/laa-maat-orchestration-dev-ecr:${CIRCLE_SHA1}"
```
Comment (venkat980) [inline .circleci/config.yml:136]: Please check the docker image name, there is no 'ecr' prefix.

File: .circleci/config.yml (hunk 2 above)
Comment (skasthuri36326) [reply .circleci/config.yml:141]: ear repo name created by terraform has ecr appended to the namespace `repo_name = "${var.namespace}-ecr"` I updated the helm values file accordingly.

Comment (skasthuri36326) [reply .circleci/config.yml]: agree. The repo name in the terraform for ecr doesn't have -ecr in the latest update.

**Review by venkat980 — APPROVED** [review]

**Pull request merged**